pkgs/data/fonts/iosevka/default.nix

```diff
@@ -1,17 +1,15 @@
-{ stdenv, lib, pkgs, fetchFromGitHub, nodejs, nodePackages, remarshal
-, ttfautohint-nox
-
-# Custom font set options.
-# See https://github.com/be5invis/Iosevka#build-your-own-style
-# Ex:
-# privateBuildPlan = {
-#   family = "Iosevka Expanded";
-#
-#   design = [
-#     "sans"
-#     "expanded"
-#   ];
-# };
+{ stdenv, lib, pkgs, fetchFromGitHub, nodejs, nodePackages, ttfautohint-nox
+  # Custom font set options.
```

Comment:
Maybe we could support both, and just check `if builtins.isAttrs privateBuildPlan then "use remarshal" else "use privateBuildPlanPath"`. I personally like using Nix to write configs and have it be translated to JSON / TOML / what have you.

Reply:
Hmm, possibly. I'd need to experiment a bit with remarshal to make sure I can create a good example config, since I think the toml format has changed since the last update.

Reply:
FWIW, here's the diff of what I had to change for the new TOML format: https://github.com/cole-h/nixos-config/commit/e46afa14cdbb139479c55726a94aa6c636fa8c54#diff-3d5395ae157ba194b854b966a6fd5b9ac3fa1c5d5e1dc606f639a18c9056cf41L90-L118